<launch>

  <arg name="camera" default="camera_lower_right" />
  <arg name="viz" default="true" />
  <arg name="grasp_detector" default="vacuum_heuristic" />
  <arg name="segmentation_method" default="clustering" />
  <arg name="camera_type" default="azure_kinect" />

  <node name="aurmr_perception" pkg="aurmr_perception" type="aurmr_perception" output="screen">
    <param name="camera_name"                   value="$(arg camera)" />
    <param name="viz"                   value="$(arg viz)" />
    <param name="segmentation_method"                   value="$(arg segmentation_method)" />
    <param name="camera_type" value="$(arg camera_type)"/>
  </node>

  <node name="grasp_detection" pkg="aurmr_perception" type="grasp_detection_server" output="screen">
    <param name="grasp_detector"                   value="$(arg grasp_detector)" />
  </node>

</launch>
